pyiron_workflow/io.py

```diff
@@ -52,7 +57,9 @@ class IO(HasStateDisplay, ABC):
     be assigned with a simple `=`.
     """
 
-    def __init__(self, *channels: Channel):
+    channel_dict: DotDict[str, OwnedType]
+
+    def __init__(self, *channels: OwnedType):
         self.__dict__["channel_dict"] = DotDict(
```

Comment:
Access via `__dict__`?

Reply:
Because we override `__setattr__`. Instead of encoding a special exception case for a particular key there, I use dict to just avoid traveling that path.